app/support_packet.go

```diff
@@ -71,32 +71,67 @@ func (a *App) generateSupportPacketYaml() (*model.FileData, string) {
 	}
 
 	// Here we are getting information regarding the database (mysql/postgres + current schema version)
-	databaseType, databaseVersion := a.Srv().DatabaseTypeAndSchemaVersion()
+	databaseType, databaseSchemaVersion := a.Srv().DatabaseTypeAndSchemaVersion()
+
+	databaseVersion, _ := a.Srv().Store().GetDbVersion(false)
 
 	uniqueUserCount, err := a.Srv().Store().User().Count(model.UserCountOptions{})
 	if err != nil {
 		return nil, errors.Wrap(err, "error while getting user count").Error()
 	}
 
+	analytics, err := a.GetAnalytics("standard", "")
+	if analytics == nil {
+		return nil, errors.Wrap(err, "error while getting analytics").Error()
+	}
+
+	elasticPostIndexing, _ := a.Srv().Store().Job().GetAllByTypePage("elasticsearch_post_indexing", 0, 2)
```

Comment:
Left this like this because I believe the job type has its own error handling. I can do it with a `GetAllByPage` but I wanted to do the last 2 jobs of each kind, so that felt like it would take more calls potentially to find all the data.